aws_doc_sdk_examples_tools/doc_gen.py

```diff
@@ -80,11 +120,12 @@ def collect_snippets(
         self.snippets = snippets
         self.errors.extend(errs)
 
-    def languages(self) -> Set[str]:
-        languages: Set[str] = set()
+    def languages(self) -> List[str]:
+        languages: List[str] = []
```

Comment:
Why the switch to List?
Although I can't find where `languages` is used so it might not matter.

Reply:
Because I needed it in a consistent order somewhere, but now I don't remember where